### DISCUSSION TITLE
Real-time frontend for simulations

### DISCUSSION THREAD

**Discussion by f0nzie** [SciML/.github#19] [Ideas]
I was reviewing my past work in computational physics and simulations, and though that given the level that Julia SciML libraries have reached, the simulation front-end from Open Source Physics  could be a match. This is how it looks.

### Initial conditions
![image](https://github.com/user-attachments/assets/8815514e-ac28-4a35-9d0f-7121d80eb8b8)

### Started
![image](https://github.com/user-attachments/assets/cb6b1db5-1d35-4683-9d2f-e360ee2c3a15)

### Stopped
![image](https://github.com/user-attachments/assets/75ff8936-f35a-45d0-9878-4542d1508a0a)

### New conditions
![image](https://github.com/user-attachments/assets/8f27a734-3a76-4e8a-a185-06e410b35f5c)

### Initialized and Run
![image](https://github.com/user-attachments/assets/b47902a4-3bd3-4614-86f4-5559b6d1692f)

If there is interest in the group I will share more examples, and could explain how the apps work if need to. Of course, I am just a user of the [Open Source Physics](https://www.compadre.org/osp/webdocs/programming.cfm?t=Overview) framework, which has the original code available for download, although is written in Java, sorely in  need of a more modern and dynamic language as Julia.

**Comment by ChrisRackauckas** [comment]
I don't think I quite understand the question.
